rtest.py

```diff
@@ -16,26 +16,45 @@
 import multiprocessing
 import time
 
+SCRIPT_VERSION = 0.1
```

Comment:
should we just remove rtest.py from this repo?

Reply:
Yes, we probably should, but currently the CI configuration doesn't copy rtest.py from the correct location in rsuite, so we can't quite yet.